### PR TITLE
Add net451 MSBuild project system back

### PR DESCRIPTION
**Pull request by troydai**

### CODE REVIEW

**Comment by troydai** [comment]
Briefly tested on Windows with NET451 OmniSharp on a csproj project. Intellisense works fine.

File: src/OmniSharp.MSBuild/ProjectFile/ProjectFileInfo.cs [hunk 1]
```diff
@@ -65,142 +70,239 @@ public class ProjectFileInfo
             var projectFileInfo = new ProjectFileInfo();
             projectFileInfo.ProjectFilePath = projectFilePath;
 
-            var properties = new Dictionary<string, string>
+            if (!PlatformHelper.IsMono)
```
Comment (DustinCampbell) [inline src/OmniSharp.MSBuild/ProjectFile/ProjectFileInfo.cs]: Was this there before?

Comment (troydai) [inline src/OmniSharp.MSBuild/ProjectFile/ProjectFileInfo.cs]: Yes it was: https://github.com/OmniSharp/omnisharp-roslyn/blob/cdfa2493df6fa4cabeb9ee44757f51319829edef/src/OmniSharp.MSBuild/ProjectFile/ProjectFileInfo.cs#L61

Comment (DustinCampbell) [inline src/OmniSharp.MSBuild/ProjectFile/ProjectFileInfo.cs]: Got it -- thanks for the link.

Comment (troydai) [inline src/OmniSharp.MSBuild/ProjectFile/ProjectFileInfo.cs]: The `else` portion are added back exactly as it was.

**Comment by DustinCampbell** [comment]
:+1: LGTM

**Comment by troydai** [comment]
You need to split the thumb up and the words after it ...

**Comment by DustinCampbell** [comment]
blech

**Comment by DustinCampbell** [comment]
:+1:

**Comment by DustinCampbell** [comment]
LGTM

**Comment by DustinCampbell** [comment]
:shipit:

**Comment by troydai** [comment]
Clearly your "LGTM" is not good enough ...

**Comment by DustinCampbell** [comment]
:+1:  LGTM  :shipit:

**Comment by troydai** [comment]
I did that ...

**Comment by DustinCampbell** [comment]
I guess @david-driscoll decided not to include me?

**Comment by troydai** [comment]
@david-driscoll not cool ...

**Comment by DustinCampbell** [comment]
Hmmm... nope, I'm in there: https://github.com/OmniSharp/omnisharp-roslyn/blob/dev/MAINTAINERS

**Comment by troydai** [comment]
or is it just slow? we can try next time when my next PR comes up.

**Comment by david-driscoll** [comment]
You're in there...

**Comment by david-driscoll** [comment]
Maybe I got the name wrong? https://github.com/OmniSharp/omnisharp-roslyn/blob/dev/MAINTAINERS

**Comment by DustinCampbell** [comment]
Nope, it's all correct. I wonder if it's something silly, like their regex requires a line break at the end of each entry or something like that.

**Pull request merged**